src/ui/CategoryFacet/CategoryFacetBreadcrumb.ts

```diff
@@ -23,12 +25,18 @@ export class CategoryFacetBreadcrumb {
 
     SVGDom.addClassToSVGInContainer(clear.el, 'coveo-facet-breadcrumb-clear-svg');
     const pathToRender = without(this.categoryValueDescriptor.path, ...this.basePath);
+    const captionLabel = pathToRender.join('/');
 
-    const title = `${this.categoryValueDescriptor.value} ${this.categoryValueDescriptor.count}`;
```

Comment:
I don't think it was meaningful having a title here with the result count. It was weird when used with the Voice Over utility